cli.js

```diff
@@ -21,9 +21,9 @@ var updateNotifier = require('update-notifier');
 var figures = require('figures');
 var arrify = require('arrify');
 var meow = require('meow');
-var chalk = require('chalk');
 var Promise = require('bluebird');
 var pkgConf = require('pkg-conf');
+var colors = require('./lib/reporters/colors');
```

Comment:
`lib/reporters` was intended to contain files of reporters only. `lib/colors.js` would be just fine as well